src/frontend/src/components/NavBar.vue

```diff
@@ -2,24 +2,28 @@
   <div class="nav-bar">
     <img
       id="nav-logo"
-      class="nav-logo"   
+      class="nav-logo"
       alt="BisonCoin logo"
       src="../assets/BisonCoin.png"
     />
     <router-link to="/account">
       <img
-        id='nav-account'
+        id="nav-account"
         class="nav-account"
         alt="Account icon"
         src="../assets/accountIcon.png"
       />
     </router-link>
-    <img
-      id="nav-logout"
-      class="nav-logout"
-      alt="Logout icon"
-      src="../assets/logoutIcon.png"
-    />
+    <div @click="logout">
+      <router-link to="/account">
```

Comment:
Don't think you need this to=/account here!

Reply:
I think you can just remove router-link from here completely actually

Reply:
Good catch!